pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/OtherIdentifiersRule.scala

```diff
@@ -0,0 +1,64 @@
+package uk.ac.wellcome.platform.merger.rules
+import uk.ac.wellcome.models.work.internal.{
+  SourceIdentifier,
+  TransformedBaseWork,
+  UnidentifiedWork
+}
+import uk.ac.wellcome.platform.merger.logging.MergerLogging
+import uk.ac.wellcome.platform.merger.rules.WorkFilters.WorkFilter
+
+/*
+ * The otherIdentifiers field is made up of all of the identifiers on the sources,
+ * except for any Sierra identifiers on Miro sources, as these may be incorrect
+ */
+object OtherIdentifiersRule extends FieldMergeRule with MergerLogging {
```

Comment:
Using partial functions like this & the field level merging, makes this _so much clearer_.